### ISSUE TITLE
[bug] √ button - Error: <method 'emit' of 'PyQt5.QtCore.py' ......

### ISSUE THREAD

**Issue by pendave** [Acly/krita-ai-diffusion#827]
![image](https://github.com/Acly/krita-ai-diffusion/assets/1331881/c1779c0b-1e84-4903-96f9-6c256753c221)
When I click the √ button：
![image](https://github.com/Acly/krita-ai-diffusion/assets/1331881/16f5ec70-c045-4af9-99a7-fb5602dbdbcf)
though the new image layer is added.

**Comment by pendave** [comment]
And I also find that the two √ buttons' function are messed up, they should function like as contrary.
![image](https://github.com/Acly/krita-ai-diffusion/assets/1331881/ada57a00-a0ed-4f02-a20c-f9e70455696b)

According to the notion:
[release-1 18 1-apply](https://github.com/Acly/krita-ai-diffusion/releases/tag/v1.18.1)

> Without regions, this is a simple way to apply a Live result as a new layer instead of replacing contents, same as previous versions.

**Comment by Acly** [comment]
Not sure I understand.
* The button on the left (just the checkmark) replaces the content
* The button on the right (with the layer beneath) adds a layer

This lines up with the tooltip description. Maybe the release notes were confusing? What I meant is that the newly introduced button follows the behavior of previous versions.

How did the error happen? Do you have some other behavior configured in settings.json?

**Comment by pendave** [comment]
I mean this button 
![image](https://github.com/Acly/krita-ai-diffusion/assets/1331881/ba1a1e6f-d568-4509-bb4e-a33b3c9fb1f0)
What I understand it is from 
> Without regions, this is a simple way to apply a Live result as a new layer instead of replacing contents, same as previous versions.
So the tooltip description should be "Create a new layer with the current result".

But the other button 
![image](https://github.com/Acly/krita-ai-diffusion/assets/1331881/a86dd514-dab0-4355-b5e2-ba90c1edac96)
is used to create a new layer first or directly replace an active layer content.
So the tooltip description should be "Copy the current reuslt to the active layer".

is the rectangle in the icon meaning "area"? If it mean "layer" then you are right.
![image](https://github.com/Acly/krita-ai-diffusion/assets/1331881/4fd44d4f-d992-435f-b5a7-ebe5d706c7a4)

**Comment by freecoderwaifu** [comment]
Everything is pretty intuitive to be honest: tooltips, icons and functionality. Checkmark icon: apply as is, applies to current layer. Checkmark icon with square below: square = layer, makes a new layer and applies. Simple as. Maybe adding a + to the second icon might give that extra clarity for some who might need it.

**Comment by pendave** [comment]
> Everything is pretty intuitive to be honest: tooltips, icons and functionality. Checkmark icon: apply as is, applies to current layer. Checkmark icon with square below: square = layer, makes a new layer and applies. Simple as. Maybe adding a + to the second icon might give that extra clarity for some who might need it.

Cool, **+** is nice for the design element.

**Comment by pendave** [comment]
```
2024-06-18 00:31:52
Traceback (most recent call last):
  File "C:\Users\pen_d\AppData\Roaming\krita\pykrita\ai_diffusion\control.py"
    layer.removed.connect(lambda: self.remove(control))
  File "C:\Users\pen_d\AppData\Roaming\krita\pykrita\ai_diffusion\control.py"
    self._layers.remove(control)
ValueError: list.remove(x): x not in list

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\pen_d\AppData\Roaming\krita\pykrita\ai_diffusion\model.py"
    return await coro
  File "C:\Users\pen_d\AppData\Roaming\krita\pykrita\ai_diffusion\model.py"
    new_input = await self._model._generate_live(self._last_input)
  File "C:\Users\pen_d\AppData\Roaming\krita\pykrita\ai_diffusion\model.py"
    cond
  File "C:\Users\pen_d\AppData\Roaming\krita\pykrita\ai_diffusion\region.py"
    control=[c.to_api(bounds) for c in list(root.control) + parent_control]
  File "C:\Users\pen_d\AppData\Roaming\krita\pykrita\ai_diffusion\region.py"
    control=[c.to_api(bounds) for c in list(root.control) + parent_control]
  File "C:\Users\pen_d\AppData\Roaming\krita\pykrita\ai_diffusion\control.py"
    layer = self.layer
  File "C:\Users\pen_d\AppData\Roaming\krita\pykrita\ai_diffusion\control.py"
    layer = self._model.layers.updated().find(self.layer_id)
  File "C:\Users\pen_d\AppData\Roaming\krita\pykrita\ai_diffusion\layer.py"
    self.update()
  File "C:\Users\pen_d\AppData\Roaming\krita\pykrita\ai_diffusion\layer.py"
    self._layers[id].removed.emit()
SystemError: <method 'emit' of 'PyQt5.QtCore.pyqtBoundSignal' objects> returned a result with an exception set

```

**Comment by Acly** [comment]
Error looks like the #823 issue

**Comment by Acly** [comment]
Error should be fixed in [v1.19.0](https://github.com/Acly/krita-ai-diffusion/releases/tag/v1.19.0)
